app/models/suggestion.py

```diff
@@ -0,0 +1,119 @@
+from .. import db
+from datetime import datetime
+import pytz
+
+
+class Suggestion(db.Model):
+    """
+    Association between a resource and potential suggestions for it
+    """
+    __tablename__ = 'suggestions'
+    id = db.Column(db.Integer, primary_key=True)
+    # resource_id is -1 for insertion
+    resource_id = db.Column(db.Integer, db.ForeignKey('resources.id'))
+    suggestion_text = db.Column(db.String(250))
+    # 0 stands for read, 1 stands for unread
+    read = db.Column(db.Boolean, default=False)
```

Comment:
How does this field if multiple users have access to the same suggestion? Will it be marked `read` for all users if one user looks at it?